data/35e/wizards_of_the_coast/supplement/dragon_magic/dm_classes.lst

```diff
@@ -14,5 +14,9 @@ CLASS:Dragonfire Adept	STARTSKILLPTS:4	CSKILL:Appraise|Bluff|Climb|Concentration
 2	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Breath Effect
 2	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Scales
 4	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Dragonkin
-6	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Damage Reduction
+6	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Damage Reduction	DR:2/magic
+8	BONUS:VAR|ScalesArmorBonus|1
```

Comment:
Will submit corrections for both this and the Breath Effect fix shortly.


Jonathon Lee Walker

On Sat, Mar 19, 2016 at 11:15 PM, Bahamut <notifications@github.com> wrote:

> In data/35e/wizards_of_the_coast/supplement/dragon_magic/dm_classes.lst
> <https://github.com/BahamutDragon/pcgen/pull/444#discussion_r56758722>:
>
> > @@ -14,5 +14,9 @@ CLASS:Dragonfire Adept	STARTSKILLPTS:4	CSKILL:Appraise|Bluff|Climb|Concentration
> >  2	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Breath Effect
> >  2	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Scales
> >  4	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Dragonkin
> > -6	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Damage Reduction
> > +6	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Damage Reduction	DR:2/magic
> > +8	BONUS:VAR|ScalesArmorBonus|1
>
> The DR should be placed in the ability and then bonused. The BONUS:VAR
> should be turned into a formula and placed into the correct ability.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/BahamutDragon/pcgen/pull/444/files/3ffc946487a8bca1f069e1e521fa4d916bdf3e6e#r56758722>
>